pkg/openthread/contrib/platform_radio.c

```diff
@@ -35,11 +36,17 @@
 #include "debug.h"
 
 #define RADIO_IEEE802154_FCS_LEN    (2U)
+#define IEEE802154_ACK_LENGTH       (5U)
+#define IEEE802154_DSN_OFFSET       (2U)
 
 static otRadioFrame sTransmitFrame;
 static otRadioFrame sReceiveFrame;
+static otRadioFrame sAckFrame;
```

Comment:
Again this is in misalignment with RIOT's coding convention. (As are the one above, that are not related to this PR.) If there is not technical reason for this, this should be renamed in order to comply with the coding convention.

Reply:
We have to adapt this file anyway in a further PR (lots of Camel Case here). Is it OK for you to leave it as is and we open a follow up right after we merge this?